src/pages/journal/journal.ts

```diff
@@ -1,11 +1,422 @@
 import { Component } from '@angular/core';
 import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { ExaminerWorkSchedule } from '../../common/domain/Journal';
 
 @IonicPage()
 @Component({
   selector: 'page-journal',
   templateUrl: 'journal.html'
 })
 export class JournalPage {
-  constructor(public navCtrl: NavController, public navParams: NavParams) {}
+  private journalJson: ExaminerWorkSchedule; // this should be populated by a call to a service
+  public journalJsonText: string; 
+  public journalSlot: any; 
```

Comment:
Weird that the interface generator didn't create one for testSlot

Reply:
I think there may still be some refactoring to be done on the schema to ensure that we get as many types as possible.